_data/simple-icons.json

```diff
@@ -1875,6 +1875,11 @@
             "hex": "535353",
             "source": "https://sourceforge.net/"
         },
+        {
+            "title": "SparkFun",
+            "hex": "E0311D",
```

Comment:
Where did you get this hex value from? I found `#E53525` in the `.ai` file in [brand-assets download](https://cdn.sparkfun.com/assets/custom_pages/3/3/4/SparkFunBrandAssets.zip) 🤔